buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -690,6 +690,13 @@ class ClusterFormationTasks {
                     env(key: 'JAVA_HOME', value: project.runtimeJavaHome)
                 }
                 node.args.each { arg(value: it) }
+                if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                    // Having no TMP on Windows defaults to C:\Windows and permission errors
+                    // Since we configure ant to run with a new  environment above, we need to set this to a dir we have access to
+                    File tmpDir = new File(node.baseDir, "tmp")
+                    tmpDir.mkdirs()
+                    env(key: "TMP", value: tmpDir.absolutePath)
```

Comment:
shouldn't we just pass through TMP? this would match the previous behavior where the environment was inherited